libs/ml/include/ml/utilities/bert_utilities.hpp

```diff
@@ -55,8 +55,8 @@ struct BERTConfig
   SizeType ff_dims           = 3072u;
   SizeType vocab_size        = 30522u;
   SizeType segment_size      = 2u;
-  DataType epsilon           = static_cast<DataType>(1e-12);
-  DataType dropout_keep_prob = static_cast<DataType>(0.9);
+  DataType epsilon           = fetch::math::Type<DataType>("0.0000000000001");
```

Comment:
maybe I'm going blind but this looks like 1e-13 to me...?

Reply:
No, you aren't )) indeed there are 13 zeroes. Corrected.